src/main/thrift/tempest.thrift

```diff
@@ -39,20 +44,21 @@ struct BidirectionalPPRParams {
 }
 
 service TempestGraphService {
-  int outDegree(1:string edgeType, 2:int id) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
-  int inDegree(1:string edgeType, 2:int id) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
+  int outDegree(1:string edgeType, 2:Node node) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
+  int inDegree(1:string edgeType, 2:Node node) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
 
-  list<int> outNeighbors(1:string edgeType, 2:int id) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
-  list<int> inNeighbors(1:string edgeType, 2:int id) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
+  list<Node> outNeighbors(1:string edgeType, 2:Node node) throws (1:InvalidNodeIdException ex1, 2:InvalidArgumentException ex2)
```

Comment:
I like this, both for consistency with other calls, and also makes it strongly typed.